.emacs.d/lisp/code/editorconfig-c.el

```diff
@@ -0,0 +1,27 @@
+;; Enable EditorConfig mode
+(editorconfig-mode +1)
+
+(defun kotct/editorconfig--disable-smart-tabs-if-indent-tabs-mode-disabled (&optional hash)
```

Comment:
Please can we shorten some of the names? This is a little excessive.That's what the docstring's for.